bench/main.cc

```
@@ -0,0 +1,173 @@
+    for (auto thread_count : num_threads) {
+      bench->Threads(thread_count);
+    }
+  }
+}
+
+int RegisterBenchmark(void (*handle)()) {
+  GetBenchmarkRegistry().push_back(handle);
+  return 0;
+}
+
+Span<const int64_t> GetInputSizes(benchmark::internal::Benchmark *bench) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)